UPDATE_PROGRAM.s

```diff
@@ -122,7 +122,7 @@
 #		V33E		VERB 33 TO SIGNAL THAT THE STATE VECTOR IS READY TO BE STORED.
 #
 #	2.  REFSMMAT (ALL DATA ENTRIES IN OCTAL)
-#		ENTRIES		DATA DEFINITITIONS				SCALE FACTORS:
+#		ENTRIES		DATA DEFINITIONS				SCALE FACTORS:
```

Comment:
Per the scan of [page 1498](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1498.jpg), this line should read:

```
#		ENTRIES:		DATA DEFINITIONS:			SCALE FACTORS:
```